modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/model.scala

```diff
@@ -222,7 +257,12 @@ object Queue {
 /**
   * A list of Steps grouped by target and instrument.
   */
-case class Sequence[A](id: String, steps: NonEmptyList[Step[A]])
+case class Sequence[A](
+  id: String,
+  steps: NonEmptyList[Step[A]],
+  // Unrooted executions
```

Comment:
This is the same case as before where an *unrooted* `Execution` represents a completed `Execution` in a yet to be completed `Step`. Once the whole `Step` is completed the *unrooted* `Execution`s are added to the `Step`.